doc/connectivity/networking/api/mqtt_sn.rst

```diff
@@ -148,7 +148,6 @@ Deviations from the standard
 
```

Comment:
For commit `mqtt_sn: implement ...`, please add `net: ` prefix to commit subject

Reply:
done